spec/std/ini_spec.cr

```diff
@@ -2,71 +2,77 @@ require "./spec_helper"
 require "ini"
 
 describe "INI" do
-  describe "parse from string" do
-    it "fails on malformed section" do
-      expect_raises(INI::ParseException, "unterminated section") do
-        INI.parse("[section")
+  describe "parse" do
+    describe "from String" do
```

Comment:
```suggestion
    context "from String" do
```

Reply:
What does it change? I don't see the improvement.

Reply:
Semantics, `describe` is typically used at top level, whereas `context` groups tests within.